### PR TITLE
Setup ModelAdmin class for each model in Challenge app

### PR DESCRIPTION
**Pull request by gauthamzz**

### CODE REVIEW

File: apps/challenges/admin.py [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+                   "enable_forum",
+                   "anonymous_leaderboard")
+    search_fields = ("title", "creator")
+
+
+@admin.register(Phase)
+class PhaseModelAdmin(TimeStampedAdmin):
+    list_display = ("name", "leaderboard_public", "challenge")
+    list_filter = ("leaderboard_public", "challenge")
+    search_fields = ("name")
```
Comment (TroJan) [inline apps/challenges/admin.py]: search_fields needs to be a tuple or list. In python when creating a tuple with single element, it is necessary to insert comma at the last, like

```
search_fields=('name',)
````

Comment (gauthamzz) [inline apps/challenges/admin.py]: done

**Comment by gauthamzz** [comment]
fixed @TroJan take a look

**Comment by TroJan** [comment]
@gauthamzz : the diff is fine. can you please squash these commits into a single one. Also for the commit message we generally prefer that you namespace it like here it can be `challenges: Setup Admin class for each model.`

File: apps/challenges/admin.py [hunk 2]
```diff
@@ -1,3 +1,19 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import Challenge, Phase
+
+
+@admin.register(Challenge)
+class ChallengeModelAdmin(TimeStampedAdmin):
```
Comment (TroJan) [inline apps/challenges/admin.py]: can you please rename it `ChallengeAdmin`

File: apps/challenges/admin.py [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+    list_display = ("title", "start_date", "end_date", "creator", "published", "enable_forum", "anonymous_leaderboard")
+    list_filter = ("creator", "published", "enable_forum", "anonymous_leaderboard")
+    search_fields = ("title", "creator")
+
+
+@admin.register(Phase)
+class PhaseModelAdmin(TimeStampedAdmin):
```
Comment (TroJan) [inline apps/challenges/admin.py]: can you rename it `PhaseAdmin`

**Comment by gauthamzz** [comment]
@TroJan and @deshraj  can you review this

**Pull request merged**